### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by jojo987N**
### Portfolio: details popup window
For this milestone :

-  implement a simple validation
- The content of the email field has to be in lower case..
- error message to the user informing them of the error
- Make sure that there are no linter errors

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -224,4 +224,43 @@ window.onload = () => {
   contents.forEach((content) => {
     observer.observe(content);
   });
+
+  //   document.querySelector("input").onchange = () => {
+  //     alert("hi")
+  //   }
```
Comment (OyePriscilla) [inline index.js]: - [ ] Please kindly remove this commented lines of code to make your work clean. 👍

File: index.js [hunk 2]
```diff
@@ -224,4 +224,43 @@ window.onload = () => {
...
+
+  const email = document.querySelector('.email');
+  const emailError = document.querySelector('.email-error');
+  const error = 'Email has to be in lower case';
+
+  const shaking = (element) => {
+    element.classList.remove('shaking');
+    // void element.offsetWidth;
```
Comment (OyePriscilla) [inline index.js]: - [ ] Please kindly remove this commented lines of code to make your work clean. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @jojo987N and @God-Logic :clap 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Contact form contains validation checking if the email is written in all lowercase ✔️
Error message display when email is not written in lowercase ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [optional] Please kindly update the **`deployment section (under Getting started header)`** of your README file, I could noticed that you already have a working deployed live link. This will make your repo informative and engaging. ℹ️ 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @jojo987N , @God-Logic 

# Approved 🍏 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
green-lantern-ok

## Highlights
- You removed the commented code pointed out by the previous reviewer ✔✔
- Email validation still works fine. :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**